k9mail/src/main/java/com/fsck/k9/activity/MessageCompose.java

```diff
@@ -1497,7 +1497,8 @@ public void onMessageBuildSuccess(MimeMessage message, boolean isDraft) {
                 message.setUid(mMessageReference.getUid());
             }
 
-            boolean saveRemotely = recipientPresenter.isAllowSavingDraftRemotely();
+            // TODO more appropriate logic here? not sure
+            boolean saveRemotely = !recipientPresenter.getCurrentCryptoStatus().shouldUsePgpMessageBuilder();
```

Comment:
This is more of a "might" be encrypted. The thing here is, the logic that decides whether a draft is uploaded or not really shouldn't live in `CryptoStatus`.